portfolio/src/main/webapp/script.js

```diff
@@ -53,6 +55,23 @@
     const titleElement = document.createElement('span');
     titleElement.innerText = text.comment;
 
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'Delete';
+    deleteButtonElement.addEventListener('click', () => {
+      deleteComment(text);
```

Comment:
I was looking at this wondering why passing in the comment text would work, then I read the entire file and realized that this is a Comment object. If you named this `commentJson` it would be easy to figure out what's happening just looking at this part by itself